crates/bevy_ui/Cargo.toml

```diff
@@ -32,4 +32,4 @@ bevy_utils = { path = "../bevy_utils", version = "0.7.0-dev" }
 stretch = "0.3.2"
 serde = { version = "1", features = ["derive"] }
 smallvec = { version = "1.6", features = ["union", "const_generics"] }
-bytemuck = { version = "1.5", features = ["derive"] }
+bytemuck = { version = "1.5", features = ["derive"] }
```

Comment:
We should revert this change to avoid noise; it looks like you're missing a trailing newline.